package.json

```diff
@@ -223,6 +224,11 @@
                 "type": "boolean",
                 "description": "Whether the app should be rebuild before start.",
                 "default": true
+              },
+              "syncAllFiles": {
```

Comment:
syncNodeModules sounds good. We have not documented this option in CLI, still it could affect existing users of VSC extension. Do you want this change in release?

@hshristov